examples/toolchains/cc_cross_osx_to_linux_amd64/README.md

```diff
@@ -1,13 +1,18 @@
-C++ With Dependencies Toolchain Example
-=======================================
+Linux C++ In Docker, Cross Compiled on MacOS
+============================================
 
-This is an example C++ project with dependencies that uses `rules_cc`.
-
-This example uses the Nix package manager to provide C++ dependencies, and as such only works with Nix installed. Demonstrating other methods of providing C++ dependencies is out of scope of this example.
+Builds a Docker image containing a C++ program cross compiled for Linux, on MacOS.
 
 # Usage
 
-To run the example with Nix, issue the following command:
+To build the image with Nix, issue the following command:
+```
+nix-shell --command 'bazel build --config=cross :hello_image_tarball'
 ```
-nix-shell --command 'bazel run --config=cross :hello'
+
+You can then do load the image into Docker with:
```

Comment:
It's also possible to just load the image with `bazel run`, [see here](https://github.com/bazelbuild/rules_docker/tree/master#using-with-docker-locally).

Reply:
I was testing on our `build02` machine, which doesn't have Docker installed, but I think it's fair to assume Docker is there for the README. Will update.